impress-org/givewp
# Feature: Add donate button caption to form settings

## Description

This PR adds a new form setting, allowing users to edit the button caption for the 'donate now' button/submit button. The applied changes will display on all 3 templates.

## Visuals

<img width="286" alt="Screen Shot 2023-08-17 at 8 58 26 AM" src="https://github.com/impress-org/givewp/assets/75056371/b851e466-de29-4667-bc0a-f1e47dffdb1f">


https://github.com/impress-org/givewp/assets/75056371/f07ca011-0868-4d31-9e17-98a2d3039962


## Testing Instructions

- Add a v3 form.
- Use the Form Settings tab and use the Donate Button setting to update the button caption.
- Verify the button caption has been updated for all 3 form templates.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205328019764976
